test/KdTreeTest.hs

```diff
@@ -59,6 +59,14 @@ prop_removePreservesInvariant points pKill =
     Kd.allSubtreesAreValid $ tree `Kd.remove` pKill
     where tree = Kd.fromList points
 
+prop_removeEdgeTest::Int->Bool
+prop_removeEdgeTest dummy = ret
```

Comment:
I'm a little confused about using a property-based test here because the code is not actually doing property-based tests.

It's conventional in Haskell to write `_` for an unused argument.